Libplanet.Explorer/Controllers/ExplorerController.cs

```diff
@@ -29,10 +29,14 @@ public class ExplorerController<T> : Controller where T : IAction, new()
         private readonly IBlockchainStore Store;
 
         public string TimestampFormat = "yyyy-MM-ddTHH:mm:ss.ffffffZ";
+        private BlockChain<T> _blockChain;
+        private static Schema _schema;
```

Comment:
I think `static` isn't mandatory. right?

Reply:
I'd heard about it verbally. `static` is needed because it should be initialized only once.